typescript/utils/src/addresses.ts

```diff
@@ -274,6 +277,11 @@ export function addressToBytes(
     new Uint8Array(),
     protocol,
   );
+  assert(
```

Comment:
nit: this check shouldn't this deep down, what if I wanted to purposefully have a zero address for burning tokens?

Reply:
Fair point. I did consider doing this in the adapters when forming txs but it would have required me to modify more than 10 classes. I'll keep it here for now but I'll watch for this in the UI's Sentry error monitoring and if we ever need to support 0 addresses I'm happy to move it.